helm/flowforge/templates/broker.yaml

```
@@ -36,7 +36,7 @@ spec:
         {{- toYaml .Values.forge.broker.podSecurityContext | nindent 8 }}
       containers:
       - name: broker
-        image: iegomez/mosquitto-go-auth
+        image: {{ .Values.forge.broker.image | default "iegomez/mosquitto-go-auth" | quote }}
```

The values file defines a default, static value. There is no need to make it redundant with the `default` function.
Also, the image value doesn't have to be quoted.

```suggestion
        image: {{ .Values.forge.broker.image }}
```